data/wanderer/wanderers start.txt

```diff
@@ -70,6 +79,13 @@ mission "First Contact: Wanderer"
 				`	"Sorry, I don't speak their language."`
 				`	"Are there any Hai here?"`
 			`	The same alien speaks again, pantomiming your ship flying into the sky and then returning, and then points to your airlock and mimics your rodent-ear gesture for the Hai. You think it is telling you to go find one of the Hai who can serve as an interpreter, and bring them back here.`
+			branch fettered
+				not "<uhai> 18: Hai Language: active"
+			`	You should probably go back to the Unfettered and speak to them about the need for a diplomat, or any way to understand these aliens. He-Yi should be able to help you with this.`
+			`	Alternatively, you could go to the Hai and get your translator that way, although He-Yi will probably respect that less.`
+				decline
+			label fettered
+			``
```

Comment:
```suggestion
			branch accept
				not "<uhai> 18: Hai Language: active"
			`	You should probably go back to the Unfettered and speak to them about the need for a diplomat, or any way to understand these aliens. He-Yi should be able to help you with this.`
			`	Alternatively, you could go to the Hai and get your translator that way, although He-Yi will probably respect that less.`
				decline
```
Assuming you need to do this to get the mission to work (something w/ `accept` immediately following a branch), but this is still a little questionable so wanted to flag it. Alternative: name the branch accept (suggested by warp).

Reply:
I dont think naming the branch accept will do anything